src/figuretype/depot.c

```diff
@@ -255,10 +255,10 @@ static int is_order_condition_satisfied(building *depot, order *order)
     }
 
     int src_amount = src->type == BUILDING_GRANARY ?
-        building_granary_resource_amount(order->resource_type, src) :
+        building_granary_resource_amount(order->resource_type, src) / 100 :
```

Comment:
Please include `city/resource.h` and use the `RESOURCE_GRANARY_ONE_LOAD` macro instead of the magic number `100`